addons/music/fnc_stopMusic.sqf

```diff
@@ -0,0 +1,25 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_stopMusic
+
+Description:
+    A function used to stop any music playing. Must have been started with CBA_fnc_playMusic, otherwise it is ignored. 
+    
+Parameters:
+    none
+
+Returns:
+    BOOL: true if music was stopped, false if already stopped
+    
+Example:
+    (begin example)
+        _bool = call CBA_fnc_stopMusic;
+    (end example)
+
+Author:
+    Fishy
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+
+playMusic ["",0];
+GVAR(track) = nil;
+true 
```

Comment:
Disagree. No function should end with an assignment operator. At least return `nil`. Returning the assignment operator has a bug in SQF.
____

```sqf
_a = call {
    _b = 1;
};
```
>`generic error in expression`

```sqf
_a = call {
    _b = 1;
    nil
};
```
>no error, _a is undefined.